python-lib/tagger.py

```diff
@@ -1,9 +1,110 @@
+"""Module to prepare the tagging of the documents
```

Comment:
Overall comment related to type hints: your code would be much more readable if you declared your methods output as type hints